service/static/js/rest_api.js

```diff
@@ -267,9 +267,7 @@ $(function () {
 
         var ajax = $.ajax({
             type: "GET",
-            url: "/api/recommendations?" + queryString,
-            contentType: "application/json",
-            data: ''
```

Comment:
This is amazing!

Reply:
Took us long time to figure out.. The only clue is the error only occurs when using the UI. No issues when using Postman or run TDD test. So has to be the JS code that are doing something differently.